service/static/js/rest_api.js

```diff
@@ -0,0 +1,262 @@
+    }
+
+    // Updates the flash message area
+    function flash_message(message) {
+        $("#flash_message").empty();
+        $("#flash_message").append(message);
+    }
+
+    function add_results_in_table(res){
+
+        $("#search_results").empty();
+            $("#search_results").append('<table class="table-striped" cellpadding="10">');
+            var header = '<tr>'
+            header += '<th style="width:10%">ID</th>'
+            header += '<th style="width:40%">User Id</th>'
+            header += '<th style="width:40%">Created On</th>'
+            header += '<th style="width:10%">Last Updated On</th></tr>'
+            $("#search_results").append(header);
+            var firstShopcart = "";
+            for(var i = 0; i < res.length; i++) {
+                var shopcart = res[i];
+                var row = "<tr><td>"+shopcart.id+"</td><td>"+shopcart.user_id+"</td><td>"+shopcart.create_time+"</td><td>"+shopcart.update_time+"</td></tr>";
+                $("#search_results").append(row);
+                if (i == 0) {
+                    firstShopcart = shopcart;
+                }
+            }
+
+            $("#search_results").append('</table>');
+            return firstShopcart;
+    }
+
+    // ****************************************
+    // Create a Shopcart
+    // ****************************************
+
+    $("#create-btn").click(function () {
+
+        var user_id = $("#user_id").val();
+        // var category = $("#pet_category").val();
+        // var available = $("#pet_available").val() == "true";
+
+        var data = {
+            "user_id": user_id
+        };
+
+        var ajax = $.ajax({
+            type: "POST",
+            url: "/shopcarts",
+            contentType: "application/json",
+            data: JSON.stringify(data),
+        });
+
+        ajax.done(function(res){
+            update_form_data(res)
+            $("#search_results").empty();
+            $("#search_results").append('<table class="table-striped" cellpadding="5">');
+            var header = '<tr>'
+            header += '<th style="width:3%">ID</th>'
+            header += '<th style="width:3%">User Id</th>'
+            header += '<th style="width:3%">Created On</th>'
+            header += '<th style="width:3%">Last Updated On</th></tr>'
+            $("#search_results").append(header);
+            var firstShopcart = "";
+            shopcart = res
+            var row = "<tr><td>"+shopcart.id+"</td><td>"+shopcart.user_id+"</td><td>"+shopcart.create_time+"</td><td>"+shopcart.update_time+"</td></tr>";
+            $("#search_results").append(row);
+                
+
+            $("#search_results").append('</table>');
+            flash_message("Success")
+        });
+
+        ajax.fail(function(res){
+            flash_message(res.responseJSON.message)
+        });
+    });
+
+
+    // ****************************************
+    // Update a shopcart
+    // ****************************************
```

Comment:
Should this be removed for now too?

Reply:
I've kept it so that I can refer to it when writing the code for shopcart items